Gradle plugin to auto install the apk on emulator?

Would be great if there was a plugin which downloaded and installed the apk prior to running Android tests. Would make set up and CI setup sooo much easier!

Actually it is also failing with same reason on emulator without playstore with api 26:

`system-images;android-26;google_apis;x86`

Reason:

```
Caused by: com.android.ddmlib.InstallException: INSTALL_FAILED_SHARED_USER_INCOMPATIBLE: Package couldn't be installed in /data/app/com.linkedin.android.testbutler-BbYyEzpMbbuCNDKyVia5pQ==: Package com.linkedin.android.testbutler has no signatures that match those in shared user android.uid.system; ignoring!
```

--- a/README.md
+++ b/README.md
@@ -69,6 +69,7 @@ Download the latest .apk and .aar via Maven:
 or Gradle:
 ```
     androidTestImplementation 'com.linkedin.testbutler:test-butler-library:2.0.0'
+    androidTestUtil 'com.linkedin.testbutler:test-butler-app:2.0.0'
 ```
 
 You can also download the apk file manually from [Bintray](https://bintray.com/linkedin/maven/test-butler-app/) if you prefer.

